src/include/workflow.h

```diff
@@ -84,6 +84,13 @@ pgmoneta_workflow_delete(struct workflow* workflow);
 struct workflow*
 pgmoneta_workflow_create_local_storage(void);
 
+/**
+ * Create a workflow for the SSH storage engine
+ * @return The workflow
+ */
+struct workflow*
+pgmoneta_workflow_create_ssh_storage(void);
```

Comment:
Create a `storage.h` with the storage functions. For now, it will include `workflow.h`